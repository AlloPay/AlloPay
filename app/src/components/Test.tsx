import React from 'react';
import { View } from 'react-native';

import { useSafes } from '@queries';
import { useSafe } from '@features/safe/safe.provider';
import { useWallet } from '@features/wallet/wallet.provider';
import { Paragraph } from 'react-native-paper';

export default () => {
  const wallet = useWallet();
  const safe = useSafe();
  const { safes } = useSafes();

  console.log({ safes: safes?.map((s) => s.id) });

  return (
    <View style={{ display: 'flex', flexDirection: 'column' }}>
      <Paragraph>Safe address: {safe.contract.address}</Paragraph>
      <Paragraph>Wallet address: {wallet.address}</Paragraph>
    </View>
  );
};
