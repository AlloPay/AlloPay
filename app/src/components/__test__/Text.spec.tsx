import Test from '@components/Test';
import { render } from '@testing-library/react-native';

describe('Test', () => {
  it('renders correctly', async () => {
    const tree = render(<Test />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
