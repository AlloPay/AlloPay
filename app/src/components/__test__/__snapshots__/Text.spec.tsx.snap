// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test renders correctly 1`] = `
<Text>
  Snapshot test!
</Text>
`;

exports[`renders correctly 1`] = `
<Text>
  Snapshot test!
</Text>
`;
